packages/video_player/ios/Classes/VideoPlayerPlugin.m

```diff
@@ -259,7 +389,8 @@ - (FlutterError* _Nullable)onCancelWithArguments:(id _Nullable)arguments {
 - (FlutterError* _Nullable)onListenWithArguments:(id _Nullable)arguments
                                        eventSink:(nonnull FlutterEventSink)events {
   _eventSink = events;
-  [self sendInitialized];
+  // TODO do we need this?
+  // [self sendInitialized];
```

Comment:
We should keep this.  It's an unnecessary breaking change.  Am I missing some reason why it's bad for us to send this?